### ISSUE TITLE
DoReFa-Net: Training Low Bitwidth Convolutional Neural Networks with Low Bitwidth Gradients

### ISSUE THREAD

**Issue by usersan** [usersan/papers#13]
## 0. 論文
Shuchang Zhou, Yuxin Wu, Zekun Ni, Xinyu Zhou, He Wen, Yuheng Zou

https://arxiv.org/abs/1606.06160

## 1. どんなもの？
1-bitではなく、k-bit量子化対応
W1A4など

## 6. 次に読むべき論文は？
